src/pptrees/prefix_graph.py

```diff
@@ -530,6 +532,12 @@ def hdl(self,out=None,full_flat=False):
         if out is not None:
             with open(out,'w') as f:
                 print(hdl,file=f)
+            # Write mapping to file
+            map_hdl = (importlib_resources.files("pptrees") / "mappings" / (mapping+'.v')).read_text()
```

Comment:
do you need the whole file in memory? since you don't do any extra processing, maybe you could use https://docs.python.org/3/library/shutil.html?